bot/exts/evergreen/emoji_count.py

```diff
@@ -61,8 +61,8 @@ def generate_invalid_embed(ctx: commands.Context) -> Tuple[discord.Embed, str]:
         msg.append(f"These are the valid categories\n```{error_comp}```")
         return embed, msg
 
-    @commands.command(name="emoji_count", aliases=["ec"])
-    async def emoji_count(self, ctx: commands.Context, *, category_query: str = None) -> None:
+    @commands.command(name="emojis_count", aliases=["ec"])
```

Comment:
My bad if i explained it badly, but what I meant was to add more aliases, not change the name. The name was fine.
Aliases like `emojis` or `emojis_count` is what i wanted to add.  Since I don't like having to use `_` when calling commands & `ec` is like a hotkey for seasoned keyboard ninjas. Having a middleground like emojis would help.